02_bubble/vinilapeddireddy/bubblesort_test.go

```diff
@@ -0,0 +1,62 @@
+package main
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+//To test the bubbleFunction with positive numbers
+func TestBubble_PositiveNumbers(t *testing.T) {
+	// Given
+	r := require.New(t)
+	var buf bytes.Buffer
+	out = &buf
```

Comment:
Setting `out` is not required for this test, as you are testing the return value of `bubble()`, not what is printed by `main()`.

Please remove these two lines, which will make the test more readable without unnecessary code.